### DISCUSSION TITLE
Cors Block When Phone app Running from IP

### DISCUSSION THREAD

**Discussion by GaryJurgens** [quasarframework/quasar#9877] [CLI - Capacitor / Cordova modes]
Hi

I seem to have the typical CORS block when my Phone app is running on the external IP (dev servers IP) but this does not happen on the Website version, running local host. 

any ideas,  research says its not a Quasar issue, but an Node.js issue.  but I cannot seem to come right. can anyone please point me in the right direction.  

App loads, but any callbacks, IE button actions are CORS blocked.

**Comment by MilosPaunovic** [comment]
If you are using Node on your web server, then you're probably using Express as well. If that is the case, check out [this](https://expressjs.com/en/resources/middleware/cors.html).

**Comment by ontwikkelfabriek** [comment]
@adolfie450 

Are you having this issue  in `dev` mode?
If so this could help:
https://quasar.dev/quasar-cli/api-proxying#introduction

**Comment by GaryJurgens** [comment]
Hi thanks for this, I am running my API in a .net core server, running on localhost: 4431. I have droped cors on the .net core server, added allow all origins, allheaders, all methods, even white listing the IPs  it's perfectly fine with browsers now no need for core unbokk extensions, how ever it's just my phone, connected via USB or not, and the android emulator.

My Desktop is printable on its up, connected to same WiFi, firewall is down.

I will try your suggestions first thing tomorrow..



Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Reply by ontwikkelfabriek** [reply]
@adolfie450 

Try running it on the phone in a non **dev** mode.

**Comment by GaryJurgens** [comment]
Morning, my apologies for the delayed reply. I will do this this morning and revert.

Thank you for your time and assistance so far.

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________
